llvm/lib/CodeGen/GlobalISel/LegalizerHelper.cpp

```diff
@@ -4920,6 +4920,8 @@ LegalizerHelper::fewerElementsVector(MachineInstr &MI, unsigned TypeIdx,
   case G_INTRINSIC_LLRINT:
   case G_INTRINSIC_ROUND:
   case G_INTRINSIC_ROUNDEVEN:
+  case G_LROUND:
+  case G_LLROUND:
```

Comment:
unrelated changes?

Reply:
The lit test is not testing this but it will complete the handling of cases in line with similar intrinsic, LRINT.